build/deploy.sh

```diff
@@ -1,14 +1,28 @@
 #!/usr/bin/env bash
 
-GETH_ARCHIVE_NAME="core-geth-${TRAVIS_OS_NAME}-$(git describe --always)"
-zip -j "$GETH_ARCHIVE_NAME.zip" build/bin/geth
 
-shasum -a 256 $GETH_ARCHIVE_NAME.zip
-shasum -a 256 $GETH_ARCHIVE_NAME.zip > $GETH_ARCHIVE_NAME.zip.sha256
+GETH_ARCHIVE_NAME="core-geth-${BUILD_OS_NAME}"
+ALLTOOLS_ARCHIVE_NAME="core-geth-alltools-${BUILD_OS_NAME}"
```

Comment:
I think we still need to get the version identifier in here, eg. `v1.12.2`. 

`core-geth-alltools-arm.zip` should be `core-geth-alltools-arm-v1.12.2.zip`

Reply:
Sorry, will add them back. I am sure I have seen somewhere the released zips being without the version.

Reply:
Resolved by 690f1ed.
You can check a build [here](https://github.com/ziogaschr/core-geth/releases/tag/v1.12.12-ghbuilds)